### ISSUE TITLE
Content updates to Cultural Heritage Collaboration section of landing page

### ISSUE THREAD

**Issue by zachjschwartz** [project-lux/lux-frontend#815]
**Problem Description:**
The LUX User Support group approved a decision to update the landing page links in the Cultural Heritage Collaboration section, currently pointing to the unit homepages, to instead direct to the record for the unit. This will encourage users to continue browsing LUX, rather than giving them an immediate prompt to visit external websites.

**Expected Behavior/Solution:**
The unit links currently listed on the landing page will now direct to internal LUX searches for the corresponding unit. This will also require several updates to the content of this section, detailed below.

**Requirements:**

- [ ] Section Header: Update "More About LUX" header to "View Collections in LUX"
- [ ] Body text: Update to better suit this section (current text is largely duplicative of "What is LUX" above)
- [ ] Update link destinations to LUX search for each unit

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - Seong
- [ ] Frontend requirements- Peter
- [ ] Are new regression tests required for QA - Sarah

**Questions**

**UAT/LUX Examples:**

https://lux-front-tst.collections.yale.edu/

**Dependencies/Blocks:**

Blocked By: Issues that are blocking the completion of the current issue.

Blocking: Issues being blocked by the completion of the current issue.

**Related Github Issues:**

- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**

- https://www.bugherd.com/projects/284041/tasks/3160

**Wireframe/Mockup:**

Current:

<img width="1684" height="846" alt="Image" src="https://github.com/user-attachments/assets/bd0c036e-7ccc-4b6e-8064-e21ef74a83c5" />

Expected:

<img width="1438" height="714" alt="Image" src="https://github.com/user-attachments/assets/51404a98-962c-4cfb-a75c-7dacc21632a4" />

**Comment by prowns** [comment]
@miketullo95 - suggested text from LUX User Support group:  _Connect with the records from Yale’s museums, archives and libraries by clicking the links below._

**Comment by zachjschwartz** [comment]
@prowns idk WHAT is going on the formatting on this ticket. Looks fine in preview but is all jumbled when changes are saved. If it's a problem I can try rewriting it from the template.

**Comment by zachjschwartz** [comment]
New guidance from accessibility team suggests that changing only the header to better reflect the contents of section would be sufficient for compliance. Will discuss at forming.

**Comment by zachjschwartz** [comment]
@prowns it's not currently but could be turned around in time. Was going to discuss this at forming but Nick Taylor's guidance was updating one of the two headings currently on that front page section. He provided these examples:

"_Search LUX Collections_
_Yale's Cultural Heritage IT Collaboration_

_or:_

_More About LUX_
_Yale’s Cultural Heritage IT Collaboration - Search the Collections_"
